_data/cryptocurrencies.yml

```diff
@@ -480,8 +480,12 @@ websites:
     url: https://www.luno.com/
     img: luno.png
     tfa:
+      - sms
+      - email
```

Comment:
```suggestion
```
The doc page does not mention email or SMS. Can you elaborate on that?

Regarding your screenshot: Does this happen only when you use a new device? It also seems to use a 4 digit code there, instead of the usual 6 digits as seen in the Google Authenticator example video.